src/dht/behaviour.rs

```diff
@@ -206,19 +286,69 @@ pub struct BillAttachmentFileRequest {
     pub bill_name: String,
     pub file_name: String,
 }
+/// Constructs a formatted string representing a file request for a bill attachment.
+///
+/// # Parameters
+///
+/// - `node_id`: A string slice representing the unique identifier of the node requesting the file.
+/// - `bill_name`: A string slice representing the name of the bill associated with the file.
+/// - `file_name`: A string slice representing the name of the file being requested.
+///
+/// # Returns
+///
+/// A `String` formatted as:
+/// ```text
+/// {node_id}_{BILL_ATTACHMENT_PREFIX}_{bill_name}_{file_name}
 
 pub fn file_request_for_bill_attachment(node_id: &str, bill_name: &str, file_name: &str) -> String {
     format!("{node_id}_{BILL_ATTACHMENT_PREFIX}_{bill_name}_{file_name}")
 }
-
+/// Constructs a formatted string representing a file request for a bill.
+/// # Parameters
+///
+/// - `node_id`: A string slice representing the unique identifier of the node requesting the file.
+/// - `bill_name`: A string slice representing the name of the bill associated with the file.
+///
+/// # Returns
+///
+/// A `String` formatted as:
+/// ```text
+/// {node_id}_{BILL_PREFIX}_{bill_name}
+///
 pub fn file_request_for_bill(node_id: &str, bill_name: &str) -> String {
     format!("{node_id}_{BILL_PREFIX}_{bill_name}")
 }
-
+/// Constructs a formatted string representing a file request for bill keys.
+/// # Parameters
+///
+/// - `node_id`: A string slice representing the unique identifier of the node requesting the file.
+/// - `bill_name`: A string slice representing the name of the bill associated with the keys.
+///
+/// # Returns
+///
+/// A `String` formatted as:
+/// ```text
+/// {node_id}_{KEY_PREFIX}_{bill_name}
+/// ```
 pub fn file_request_for_bill_keys(node_id: &str, bill_name: &str) -> String {
     format!("{node_id}_{KEY_PREFIX}_{bill_name}")
 }
 
+/// This function takes a string representing an inbound file request, splits it into parts,
+/// and categorizes the request into different types: `Bill`, `BillKeys`, or `BillAttachment`.
+///
+/// # Parameters
+///
+/// - `request`: A string slice representing the inbound file request to be parsed.
+///
+/// # Returns
```

Comment:
same here - this returns the parsed inbound file request

In case you used an AI to help create this - generally, I consider using LLMs for generating docs as totally fine, but you always have to work over it, that it makes sense and becomes useful.

For example, some of these descriptions are way too specific to the code they describe and if that's the case, I might as well just read the code. :sweat_smile: 

Conceptually describing what happens inside the function, mentioning error cases and preconditions would be the perfect level of detail we're looking for. :+1: